src/toggle/toggle.tsx

```diff
@@ -122,6 +185,177 @@ export const Toggle = ({
         );
     };
 
+    const renderCompositeOptionSection = () => {
+        if (!compositeOptionSection) {
+            return null;
+        }
+        const ChildrenisFinalItem = !collapsible || removable;
+        return (
+            <Children
+                $selected={showMore}
+                $isNotFinalItem={
+                    ChildrenisFinalItem ? !ChildrenisFinalItem : selected
+                }
```

Comment:
found this confusing as the variable is double negated. generally for booleans, it's more readable if [the naming is affirmative](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)